lib/tasks/users.rake

```diff
@@ -12,6 +12,31 @@ namespace :users do
   task update_user_roles_to_standard: :environment do
     update_user_roles
   end
+
+  desc "Delete user (dry run)"
+  task :delete_user_dry_run, %i[user_id] => :environment do |_, args|
+    usage_message = "usage: rake delete_user_dry_run[<user_id>]".freeze
+    abort usage_message if args[:user_id].blank?
+
+    ActiveRecord::Base.transaction do
+      user = User.find(args[:user_id])
```

Comment:
Minor - but do you reckon it's worth extracting the small amount of main logic into a private function shared between the dry-run and non-dry run task. If we want to change or adapt this it's helpful to have the code shared.

Reply:
Good suggestion, have updated!